server/chat-commands/info.ts

```diff
@@ -2564,34 +2564,14 @@ export const commands: ChatCommands = {
 
 	'!code': true,
 	code(target, room, user) {
-		// XXX: target is trimmed by Chat#splitMessage. Let's not add another
-		// awful hack like ! or help command keys for whether or not the target
-		// is raw for now.
-		target = this.message.substr(this.cmdToken.length + this.cmd.length + +this.message.includes(' ')).trimRight();
```

Comment:
Why did you remove this?

Reply:
This is relevant for leading whitespace (lines starting with the empty space `" "`)

Reply:
i just moved it downwards a bit

Reply:
Oh, you moved it down and removed the comment. Why?

Reply:
Comment didn't seem necessary - as for the trimming, i originally removed it, then realized it was necessary and put it back in slightly lower.